daffodil-test/src/test/resources/org/apache/daffodil/section14/sequence_groups/SequenceGroup.tdml

```diff
@@ -1406,9 +1406,9 @@
       <tdml:documentPart type="text"><![CDATA[145,2]]></tdml:documentPart>
     </tdml:document>
     <tdml:errors>
-      <tdml:error>choice</tdml:error>
-      <tdml:error>hidden group</tdml:error>
-      <tdml:error>outputValueCalc</tdml:error>
+      <tdml:error>Schema Definition Error</tdml:error>
+      <tdml:error>hidden choice</tdml:error>
+      <tdml:error>must be fully defaultable or define dfdl:outputValueCalc</tdml:error>
```

Comment:
Better to be less specific about the error messages so that minor changes to them may not break the test.
I would just have the error checks look for "defaultable" and "outputValueCalc" somewhere in the message.